include/libtorrent/alert_types.hpp

```diff
@@ -1843,7 +1843,7 @@ namespace libtorrent
 		// interpret these values throughout the process' runtime.
 		//
 		// For more information, see the session-statistics_ section.
-		std::uint64_t values[counters::num_counters];
+		std::int64_t values[counters::num_counters];
```

Comment:
the reason I hesitate is that I seem to recall that the session_view (as part of client_test) specifically relies on mutating the counters it receives. I'll take a look if that's the case